lib/services/local/trust.go

```diff
@@ -348,6 +452,16 @@ func (s *CA) UpdateUserCARoleMap(ctx context.Context, name string, roleMap types
 	return nil
 }
 
+// activeKey builds the active key variant for the supplied ca id.
+func activeKey(id types.CertAuthID) []byte {
+	return backend.Key(authoritiesPrefix, string(id.Type), id.DomainName)
+}
+
+// inactiveKey builds the inactive key variant for the supplied ca id.
+func inactiveKey(id types.CertAuthID) []byte {
+	return backend.Key(authoritiesPrefix, deactivatedPrefix, string(id.Type), id.DomainName)
+}
+
```

Comment:
These are free functions in `lib/services/local`, we should call them `(in)activeCAKey` or something like that.